docs/src/rules/no-unused-vars.md

```diff
@@ -265,7 +265,7 @@ Examples of **correct** code for the `{ "argsIgnorePattern": "^_" }` option:
 ::: correct
 
 ```js
-/*eslint no-unused-vars: ["error", { "argsIgnorePattern": "^_" }]*/
+/*eslint no-unused-vars: ["error", { "args": "all", "argsIgnorePattern": "^_" }]*/
```

Comment:
Why was this change required? I believe the example was correct earlier as well.